x/lockup/keeper/internal/events/emit_test.go

```diff
@@ -0,0 +1,264 @@
+package events_test
```

Comment:
This entire test file seems kinda useless

Reply:
This makes sense, the entire boilerplate tests definitely seems like an overkill. I will remove this so that we can test messages that emit these events directly in `msg_server_test.go`

Reply:
In my eyes, the goal at this level of abstraction is to test that attributes are set correctly. This is achieved by the tests here

Then, it is possible to simply focus that a certain event kind is emitted at the keeper/message server layer. Given that we have already tested the attributes here in `emit_test.go`, we can reduce the boilerplate in tests for keeper and simply call for testing the fact that an event has been emitted:
https://github.com/osmosis-labs/osmosis/blob/a1dc69413fd908810dbafe150967e814b796897b/app/apptesting/events.go#L7

Otherwise, we would have to validate the attributes in keeper and message server tests which goes against the separation of concerns